pkg/cli/images.go

```diff
@@ -173,3 +215,33 @@ func newImageContainerList(image apiv1.Image, containers map[string]v1.Container
 
 	return imageContainers
 }
+func parseRepo(tag string) string {
+	parts := strings.Split(tag, ":")
+	switch len(parts) {
+	case 3:
+		return parts[0] + ":" + parts[1]
+	case 2:
+		return parts[0]
+	}
+	return "docker.io"
+}
+func parseTag(tag string) string {
+	parts := strings.Split(tag, ":")
+	switch len(parts) {
+	case 3:
+		return parts[2]
+	case 2:
+		return parts[1]
+	}
+	return "<none>"
+}
+func includesTag(name string) bool {
+	return len(strings.Split(name, ":")) == 2
+}
+
+func includesRepoAndTag(name string) bool {
+	if len(strings.Split(name, "/")) == 2 {
+		return len(strings.Split(strings.Split(name, "/")[1], ":")) == 2
+	}
+	return false
+}
```

Comment:
Not blocking; Just a thought, but if there were to be a bug in these functions some logic could get seriously messed up. It may be worth adding some unit tests that validate their logic so functions using them can feel safe.

Reply:
agreed, added some unit tests in latest commit